Buildings/Fluid/HeatExchangers/WetCoilEffectivenessNTU.mo

```diff
@@ -0,0 +1,583 @@
+within Buildings.Fluid.HeatExchangers;
+model WetCoilEffectivenessNTU
+  "Heat exchanger with effectiveness - NTU relation and with moisture condensation"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    redeclare replaceable package Medium2 = Buildings.Media.Air);
+  extends Buildings.Fluid.Interfaces.FourPortFlowResistanceParameters(
+    final computeFlowResistance1=true,
+    final computeFlowResistance2=true);
+
+  import con = Buildings.Fluid.Types.HeatExchangerConfiguration;
+  import flo = Buildings.Fluid.Types.HeatExchangerFlowRegime;
+
+  parameter Modelica.SIunits.ThermalConductance UA_nominal(min=0)
+    "Thermal conductance at nominal flow, used to compute heat capacity"
```

Comment:
"heat capacity" is ambiguous as one could think of cp.
I suggest to simply remove it.
I think the important part that UA must correspond to *fully dry* conditions is missing. (Even if it is implied by the definition of the thermal conductance, a user still might be confused.)

Reply:
@kim1077 : can you please address this